### DISCUSSION TITLE
Can I connect a landline to this version of Asterisk?

### DISCUSSION THREAD

**Discussion by shaharatavenstein** [TECH7Fox/asterisk-hass-addons#193] [Ideas]
Hi,

I was wondering if I can connect a landline using a bridge such as https://www.amazon.com/dp/B06XW1BQHC/?coliid=IGWD13CE52ADB&colid=39C9ULA7FP7WW&psc=1&ref_=lv_ov_lig_dp_it to connect my landline to HA?

**Comment by TECH7Fox** [comment]
Yes you can get a bridge for that, but not with that device. The ht801 only has a FXS port, and to connect to a landline you need a FXO port.
